xml/http_cookies.xml

```diff
@@ -127,10 +127,11 @@
   <fingerprint pattern="^NSC_(?:AAAC|CERT|DLGE|EPAC|TASS|TEMP|TMA[APS]|PERS)=.*">
     <description>Citrix NetScaler</description>
     <example>NSC_AAAC=xyz;</example>
-    <param pos="0" name="os.vendor" value="Citrix"/>
-    <param pos="0" name="os.family" value="NetScaler"/>
-    <param pos="0" name="os.device" value="Network Management Device"/>
-    <param pos="0" name="os.product" value="NetScaler"/>
+    <param pos="0" name="service.vendor" value="Citrix"/>
```

Comment:
This removes the OS assert, it should not. This should probably assert HW too.

Reply:
I can assert that as FreeBSD

Reply:
It should be asserted as what it was before, not FreeBSD (its a custom OS)

Reply:
This one in particular is a big deal, as we use the os and hw fields to set the device type and hardware type. Folks trying to find their possibly-exploitable NetScalers need this fingerprint to remain how it was (from our perspective).

Reply:
Yup, I can definitely sort this out. I'll try to review everywhere Netscaler exists and try to make it consistent.

Reply:
@hdm - I'm going to sort this out this morning. RE: hardware assertions - there are virtual Netscalers. I'm not sure if we have an established policy on how to handle `hw.*` vs virtual machines. I'm leaning towards going ahead an using it here. Any thoughts?